packages/x-data-grid/src/models/gridStateCommunity.ts

```diff
@@ -31,7 +30,7 @@ import type { GridColumnResizeState } from '../hooks/features/columnResize';
  * The state of `DataGrid`.
  */
 export interface GridStateCommunity {
-  theme: Theme;
+  isRtl: boolean;
```

Comment:
I think this would be a breaking change, the theme selector is public.

Reply:
Good question that I don't know the answer to. 🙈 
Can you confirm this @cherniavskii @MBilalShafi?

Reply:
Nevermind, it doesn't have `@ignore` but it's also not exported from the package, this looks fine.